src/app/education-feature/education/education.component.ts

```diff
@@ -9,11 +17,80 @@ import { faPencil } from '@fortawesome/free-solid-svg-icons';
 })
 export class EducationComponent implements OnInit {
 
+  educations$: Observable<EducationPagination | null> = this.educationStore.select(selectMyEducations);
+  myEducations$: Observable<Education[]> = this.educations$.pipe(
+    filter((education): education is EducationPagination => education !== undefined),
+    map(educations => educations?.content));
   faPencil = faPencil
+  faTrashCan = faTrashCan
+  faXmark = faXmark
+  faTriangleExclamation = faTriangleExclamation;
+
+  active: boolean = false;
+  showModal: boolean = false;
+
+  currentPage = 0;
+  pageAmountSub$: Observable<number> = this.educations$.pipe(
+    filter((education): education is EducationPagination => education !== undefined),
+    map(educations => educations?.totalPages));
+  pageAmount: number = 0;
+
+  educationForm = this.fb.group({
+    id: [0, Validators.required],
+    diploma: ['', Validators.required],
+    description: ['', Validators.required],
+    school: ['', Validators.required],
+    fieldOfStudy: ['', Validators.required],
+    country: ['', Validators.required],
+    website: ['', Validators.required],
+    startDate: ['', Validators.required],
+    endDate: ['', Validators.required],
+    active: [false, Validators.required],
+    candidateId: ['', Validators.required]
+  })
 
-  constructor() { }
+  constructor(private router: Router, private fb: FormBuilder, private educationStore: Store<{ educations: EducationPagination }>) {
+  }
 
   ngOnInit(): void {
+    this.educationStore.dispatch(loadEducations({page: this.currentPage}));
+    this.myEducations$.subscribe()
```

Comment:
Subscriptions without a pipe(take()) can lead to 'memory leaks' and is a bad practise.